common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1423,7 +1423,7 @@ def save_problem(self, data):
         self.track_function_unmask('save_problem_success', event_info)
         msg = _("Your answers have been saved.")
         if not self.max_attempts == 0:
-            msg = _("Your answers have been saved but not graded. Click 'Check' to grade them.")
+            msg = _("Your answers have been saved but not graded. Click '" + self.check_button_name() +  "' to grade them.")
```

Comment:
You are correct, this does not meet our i18n rules. Please see http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html for details.